use-timescale/security/read-only-role.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Read-only role
+excerpt: Create a role to allow read-only access to your database
+product: [cloud, mst, self_hosted]
+keywords: [client credentials, accounts, users, members, read-only, security]
+tags: [authentication, credentials, members, security]
+---
+
+# Read-only user
+
+You can create a role that provides read-only access to your database.
+
+<Highlight type="important">
+Adding a read-only user role to your database does not provide resource
+isolation. If you require the security of a read-only user, as well as resource
+isolation, you can create a read replica instead. For more information, see the
+[read scaling](/use-timescale/latest/ha-replicas/read-scaling/)
+section.
+</Highlight>
+
+## Create a read-only user
+
+You can create a read-only user to provide limited access to your database.
+
+<Procedure>
+
+### Creating a read-only user
+
+1.  At the psql prompt, create the new role:
+
+    ```sql
+    CREATE ROLE readaccess;
+    ```
```

Comment:
we can add that you must connect as `tsdbadmin`, otherwise we get` ERROR:  permission denied to create role`. 
As far as I know, 95-99% of people use this role to perform administration of their DB. The rest create a new role and reassign admin rights to it.